docs/about/faq.md

```diff
@@ -134,3 +134,13 @@ in their output of `tailscale status`. Traffic is still filtered according to th
 ping` which is always allowed in either direction.
 
 See also <https://tailscale.com/kb/1087/device-visibility>.
+
+## My policy is stored in the database and Headscale refuses to start due to an invalid policy. How can I recover?
+
+Headscale checks if the policy is valid during startup and refuses to start if it detects an error. The error message
+indicates which part of the policy is invalid. Follow these steps to fix your policy:
+
+- Dump the policy to a file: `headscale policy get --bypass-grpc-and-access-database-directly > policy.json`
```

Comment:
_if_ the user has split their config to a minimal cli config for the command controlling headscale, then they need to use the server config here so we get the database config. Not sure how widespread it is, but I think nixos is configured in this way.